### ISSUE TITLE
.json File Content Disappearing 

### ISSUE THREAD

**Issue by SaucedBox** [JannisX11/blockbench#708]
**THIS ISSUE IS FIXED. PLEASE DELETE THIS**

When I exported my model as a .json file, I put it in the game. It comes up as a black and purple cube. So I go into the other .json file and all the code was right. Then I open my model's .json file. There was nothing there. My model was gone, and the textures where missing even though I remember putting the textures in the correct file location. 

I also have no credits in the code, I am running the newest version, and I have uninstalled Blockbench and reinstalled. This is clearly and issue that needs to be fixed.

**Comment by JannisX11** [comment]
Regarding missing elements, please check if your elements are set to export. To do this, press "Toggle More Options" above the outliner and check if the "Export" toggle shows a white camera icon on all cubes. Regarding the credit line, check in the Blockbench settings under "Export" if a credit comment is defined. If this field is left empty, no credit line will be exported. Also check if you exported your model to the right location.

If none of this helps, please provide a clear step-by-step guide on how to reproduce the issue.

**Comment by SaucedBox** [comment]
Hello. This is the person that has an issue with model content
disappearing.

I have tried your advice, and all the things you said where already done. I
have no credits in my code, and all of the cubes have export turned on. I
have no clue how this glitch happened. I just got on to Blockbench one day,
and I tried to export my model and it happened. I have no idea how it
happened, or if I have a messed up setting, or I’m doing something
completely wrong.

Thank you for your time.
